oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -219,18 +268,21 @@ static inline void mca_spml_ucx_cache_mkey(mca_spml_ucx_ctx_t *ucx_ctx,
                                            sshmem_mkey_t *mkey, uint32_t segno, int dst_pe)
 {
     ucp_peer_t *peer;
+    spml_ucx_cached_mkey_t *ucx_cached_mkey;
 
     peer = &(ucx_ctx->ucp_peers[dst_pe]);
-    mkey_segment_init(&peer->mkeys[segno].super, mkey, segno);
+    mca_spml_ucx_ep_mkey_get(peer, segno, &ucx_cached_mkey);
```

Comment:
```suggestion
    rc = mca_spml_ucx_ep_mkey_get(peer, segno, &ucx_cached_mkey);
    if (rc) { ... }
```

Probably need to make this function to return int